shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -227,6 +227,13 @@ object JsonLdUtils {
     fun expandValueAsListOfMap(value: Any): List<Map<String, List<Any>>> =
         value as List<Map<String, List<Any>>>
 
+    fun expandGeoPropertyKey(propertyKey: String): String =
```

Comment:
do you use this function somewhere?

Reply:
No I don't use this function. I created it because I thought it might be useful later.

Reply:
https://fr.wikipedia.org/wiki/YAGNI